### ISSUE TITLE
React Native Support

### ISSUE THREAD

**Issue by urbanappetite** [chute/turbine#4]
Hey,

I've gotten turbine running on React Native with a few minor amends but I'm waiting on your latest changes described in another issue to send it as a pull request.

Have you idea when you're looking to push it to GitHub? I'm using it in a production app and I'm eager to help out where needed.

**Comment by petrbela** [comment]
Hey Luke,That's great!




We're probably going to push new stuff early next week.





I think you can create a PR right away if you have it publishable.

**Comment by sventschui** [comment]
@urbanappetite Could you please submit the PR, so we can get it working for us?
